example_app/src/lib.rs

```diff
@@ -13,13 +13,24 @@ use eframe::wasm_bindgen::{self, prelude::*};
 /// You can add more callbacks like this if you want to call in to your code.
 #[cfg(target_arch = "wasm32")]
 #[wasm_bindgen]
-pub fn start(canvas_id: &str) -> std::result::Result<(), eframe::wasm_bindgen::JsValue> {
+pub fn start(canvas_id: &str) {
```

Comment:
Another area I'm unsure about. We seem to have two entrypoints for WASM; the one that is added in this commit, and the existing one in `lib.rs`. Should only one be required?

Also, removing the return type is just to get it to compile. Not sure if this is the right thing to do here just yet.